tests/pipelines/test_predict_step.py

```diff
@@ -8,5 +8,5 @@
 
 
 def test_always_pass(tmp_pipeline_root_path, tmp_pipeline_exec_path):
-    assert tmp_pipeline_exec_path != tmp_pipeline_exec_path
+    assert tmp_pipeline_root_path != tmp_pipeline_exec_path
```

Comment:
Just to confirm, this never ran before and hence never failed?

Reply:
Correct.  I noticed it when I finally ran it that it was broken 🙂

Reply:
Yay!